src/stores/Binding/slices/ToggleDisable.ts

```diff
@@ -0,0 +1,113 @@
+import type { BindingState } from 'src/stores/Binding/types';
+import type { NamedSet } from 'zustand/middleware';
+
+import produce from 'immer';
+import { isBoolean } from 'lodash';
+
+import { hasLength } from 'src/utils/misc-utils';
+
+export type ToggleDisableUpdate = {
+    uuid: string;
```

Comment:
The generic nature of `uuid` reminds me of `id`. Should this property be renamed to `bindingUUID` to clarify the state to which the UUID corresponds?